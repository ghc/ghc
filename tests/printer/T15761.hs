{-# LANGUAGE TypeFamilies, TypeOperators #-}

data family (a + b) c d

data instance (Int + Bool) Double = Float
