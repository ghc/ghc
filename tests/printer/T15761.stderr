T15761.hs:5:1: error: [GHC-83865]
    • Expecting one more argument to ‘(Int + Bool) Double’
      Expected a type, but ‘(Int + Bool) Double’ has kind ‘* -> *’
    • In the data family instance declaration for ‘+’

